src/components/list/ItemsList.tsx

```diff
@@ -69,22 +107,25 @@ export default class ItemsList<T extends ListItem> extends React.Component<Props
     );
   }
 
-  private onSelectItem(item: T) {
-    const { selectedIds } = this.state;
-    const id = item.id;
+  private onSelectItem(item: T): void {
+    const { selectedItems } = this.state;
+    const { onSelect, selectionMode } = this.props;
+    const itemSelectedCallback = onSelect ? () => onSelect([...this.state.selectedItems.values()]) : null;
```

Comment:
```suggestion
    const itemSelectedCallback = onSelect ? () => onSelect([...this.state.selectedItems.values()]) : () => {};
```

null is not a function.